src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
```

Comment:
Avoid putting all behavior into a single interface if the methods are conceptually different. The `Drawable` and `AreaCalculator` interfaces should be separate as their methods serve different purposes.